docker-compose.yaml

```diff
@@ -0,0 +1,20 @@
+version: '2.1'
+services:
+  batch-cloud:
+    #restart: "always"
+    network_mode: host
+    image: openmrs-fhir-analytics/batch-job:latest
```

Comment:
nit: Does specifying `image` here has any benefits? I was a little bit confused first as I thought we are fetching the image from somewhere, but IIUC we are simply building from the Dockerfile, right? If you want to keep it, maybe move it after `build`?

Reply:
Noted!